README.md

```diff
@@ -1,22 +1,31 @@
 # rokt-demo-android
 
-Rokt Demo application is a sample app built to showcase Rokt SDK functionality.
+Rokt Demo application is a sample app built to showcase Rokt mobile SDK functionality. The purpose of this app is to showcase the functionality that Rokt provides in-app. The app features multiple pages with different placement examples to demonstrate the functionality of the Rokt mobile SDK.
+
+## Resident Experts
+- Sahil Suri - sahil.suri@rokt.com
+- Danial Motahari - danial.motahari@rokt.com
 
```

Comment:
Is it possible to add the build status like what we have [here](https://github.com/ROKT/sdk-android-source).
You need to visit buildKite for this one.